modules/nf-core/sgdemux/main.nf

```diff
@@ -0,0 +1,49 @@
+process SGDEMUX {
+    tag "$meta.id"
+    label 'process_high'
+
+    conda "bioconda::sgdemux=1.1.0"
+    if (workflow.profile.tokenize(',').intersect(['Singularity', 'Docker']).size() >= 1) {
+        exit 1, "SGDEMUX module does not support Singularity or Docker. Please use Conda / Mamba instead."
+    }
+    // TODO check sgdemux is not on singularity/docker (google says no) double check with Nils
```

Comment:
Having `sgdemux` in `bioconda` will make a docker container accessible on `quay.io`.  See https://bioconda.github.io/recipes/sgdemux/README.html#package-sgdemux, in particular:
`quay.io/biocontainers/sgdemux:<tag>`

For singularity, we'll need a `Dockerfile` ([see requirement (5)](https://nf-co.re/developers/modules#software-requirements)).  Feel free to ask the team how to create a dockerfile (or look for examples in other modules in nf-core/modules)

Reply:
Feel free to ping the nf-core folks on slack for help on this too!

Reply:
I see `sgdemux` on `quay.io`, and I will make changes to line 12 in the next commit to reflect that. I am happy to create a `Dockerfile` for `sgdemux`. I did not know that was up to us

Reply:
TODO: Make `Dockerfile` and update lines 9-12.